app/services/gpturk_service.rb

```diff
@@ -0,0 +1,12 @@
+module GpturkService
```

Comment:
Can you include tests for this service that include handling an error from the `RestClient.post` call? Thanks!

Reply:
Adding an entire suite for this module, apologies!